src/framework/components/model/data.js

```diff
@@ -20,6 +20,7 @@ pc.extend(pc, function() {
         this.lightmapped = false;
         this.lightmapSizeMultiplier = 1;
         this.isStatic = false;
+        this.batchGroupId = -1;
```

Comment:
Should be -1 or null if no batch is applied?

Reply:
It's always a number, so I thought -1 is fine for a non-value.

Reply:
Actually good point as this is edited by the editor... might need to be something different. I'll need to have a think